code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -670,11 +670,10 @@
 		SSquirks.AssignQuirks(new_spawn, new_spawn.client, TRUE, TRUE, null, FALSE, new_spawn)
 		new_spawn.AddElement(/datum/element/ghost_role_eligibility, free_ghosting = TRUE)
 		new_spawn.AddElement(/datum/element/dusts_on_catatonia)
-		new_spawn.AddElement(/datum/element/dusts_on_leaving_area,list(A.type,/area/hilbertshotel))
+		new_spawn.AddElement(/datum/element/dusts_on_leaving_area,list(A.type,/area/hilbertshotel | /area/centcom/holding/cafe | /area/centcom/holding/cafebuild | /area/centcom/holding/cafewar))
```

Comment:
these should just be commas, not |s, the | is for putting two lists together while this is all in one list

Reply:
i should note that this is runtiming, which breaks everything below this line

Reply:
```suggestion
		new_spawn.AddElement(/datum/element/dusts_on_leaving_area,list(A.type,/area/hilbertshotel, /area/centcom/holding/cafe, /area/centcom/holding/cafebuild, /area/centcom/holding/cafewar))
```